### ISSUE TITLE
Disable Pause on Focus Loss option is not working

### ISSUE THREAD

**Issue by MelaDoZer** [Lyall/DDDAFix#1]
I edited my copy of ini file like this and this setting didn't work.
```
[Disable Pause on Focus Loss]
; Set to true to stop the game from pausing when alt+tabbed.
Enabled = true
```

I have a solution to this problem.
I have checked dllmain.cpp and found the next lines. Pay attention to missing space between "Disable" and "Pause".
`inipp::get_value(ini.sections["DisablePause on Focus Loss"], "Enabled", bDisablePauseOnFocusLoss);`

Then I changed this line of ini file to correct naming and this setting works correctly now.
```
[DisablePause on Focus Loss]
; Set to true to stop the game from pausing when alt+tabbed.
Enabled = true
```

So it's not a big deal and I think it will be easy to fix 👍 Thanks a lot for this modification!

**Comment by Lyall** [comment]
Nice catch. Should be fixed in the next version.

**Comment by Lyall** [comment]
Fixed in v0.8.1
